`test_set_get_group_image` is flaky

Here is the log of failure:
```
2023-10-09T01:58:32.4729761Z =================================== FAILURES ===================================
2023-10-09T01:58:32.4730264Z ___________________________ test_set_get_group_image ___________________________
2023-10-09T01:58:32.4780109Z [gw3] linux -- Python 3.7.17 /home/runner/work/deltachat-core-rust/deltachat-core-rust/python/.tox/py/bin/python
2023-10-09T01:58:32.4780438Z 
2023-10-09T01:58:32.4780663Z acfactory = <deltachat.testplugin.ACFactory object at 0x7fc0cef65d90>
2023-10-09T01:58:32.4781138Z data = <deltachat.testplugin.data.<locals>.Data object at 0x7fc0cef65c50>
2023-10-09T01:58:32.4781600Z lp = <deltachat.testplugin.lp.<locals>.Printer object at 0x7fc0cef65b90>
2023-10-09T01:58:32.4782284Z 
2023-10-09T01:58:32.4782460Z     def test_set_get_group_image(acfactory, data, lp):
2023-10-09T01:58:32.4782818Z         ac1, ac2 = acfactory.get_online_accounts(2)
2023-10-09T01:58:32.4783093Z     
2023-10-09T01:58:32.4783368Z         lp.sec("create unpromoted group chat")
2023-10-09T01:58:32.4783695Z         chat = ac1.create_group_chat("hello")
2023-10-09T01:58:32.4783977Z         p = data.get_path("d.png")
2023-10-09T01:58:32.4784218Z     
2023-10-09T01:58:32.4784507Z         lp.sec("ac1: set profile image on unpromoted chat")
2023-10-09T01:58:32.4784845Z         chat.set_profile_image(p)
2023-10-09T01:58:32.4785172Z         ac1._evtracker.get_matching("DC_EVENT_CHAT_MODIFIED")
2023-10-09T01:58:32.4785499Z         assert not chat.is_promoted()
2023-10-09T01:58:32.4785730Z     
2023-10-09T01:58:32.4786045Z         lp.sec("ac1: send text to promote chat (XXX without contact added)")
2023-10-09T01:58:32.4794469Z         # XXX first promote the chat before adding contact
2023-10-09T01:58:32.4794903Z         # because DC does not send out profile images for unpromoted chats
2023-10-09T01:58:32.4795239Z         # otherwise
2023-10-09T01:58:32.4795577Z         chat.send_text("ac1: initial message to promote chat (workaround)")
2023-10-09T01:58:32.4795933Z         assert chat.is_promoted()
2023-10-09T01:58:32.4796219Z         assert chat.get_profile_image()
2023-10-09T01:58:32.4796467Z     
2023-10-09T01:58:32.4796733Z         lp.sec("ac2: check that initial message arrived")
2023-10-09T01:58:32.4797379Z         ac2.create_contact(ac1).create_chat()
2023-10-09T01:58:32.4797721Z         ac2._evtracker.get_matching("DC_EVENT_MSGS_CHANGED")
2023-10-09T01:58:32.4798001Z     
2023-10-09T01:58:32.4798277Z         lp.sec("ac1: add ac2 to promoted group chat")
2023-10-09T01:58:32.4798601Z         chat.add_contact(ac2)  # sends one message
2023-10-09T01:58:32.4798865Z     
2023-10-09T01:58:32.4799129Z         lp.sec("ac1: send a first message to ac2")
2023-10-09T01:58:32.4799466Z         chat.send_text("hi")  # sends another message
2023-10-09T01:58:32.4799771Z         assert chat.is_promoted()
2023-10-09T01:58:32.4799995Z     
2023-10-09T01:58:32.4800271Z         lp.sec("ac2: wait for receiving message from ac1")
2023-10-09T01:58:32.4800628Z         msg1 = ac2._evtracker.wait_next_incoming_message()
2023-10-09T01:58:32.4800977Z >       assert msg1.is_system_message()  # Member added
2023-10-09T01:58:32.4801299Z E       AssertionError: assert False
2023-10-09T01:58:32.4811016Z E        +  where False = <bound method Message.is_system_message of <Message incoming sys=False 'hi' id=12 sender=10/7dzs8@ci.testrun.org chat=13/hello>>()
2023-10-09T01:58:32.4812073Z E        +    where <bound method Message.is_system_message of <Message incoming sys=False 'hi' id=12 sender=10/7dzs8@ci.testrun.org chat=13/hello>> = <Message incoming sys=False 'hi' id=12 sender=10/7dzs8@ci.testrun.org chat=13/hello>.is_system_message
2023-10-09T01:58:32.4812639Z 
2023-10-09T01:58:32.4812794Z tests/test_1_online.py:1935: AssertionError
2023-10-09T01:58:32.4813234Z ----------------------------- Captured stdout call -----------------------------
2023-10-09T01:58:32.4813549Z CACHE HIT for 7dzs8@ci.testrun.org
2023-10-09T01:58:32.4814212Z [acsetup] 0.013 added already configured account <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac1/dc.db> 7dzs8@ci.testrun.org
2023-10-09T01:58:32.4814689Z CACHE HIT for m25dm@ci.testrun.org
2023-10-09T01:58:32.4815351Z [acsetup] 0.025 added already configured account <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac2/dc.db> m25dm@ci.testrun.org
2023-10-09T01:58:32.4815827Z bringing accounts online
2023-10-09T01:58:32.4819722Z wait_all_configured finds accounts= {<Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac1/dc.db>: 'CONFIGURED', <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac2/dc.db>: 'CONFIGURED'}
2023-10-09T01:58:32.4821096Z 6.30 [events-ac1] INFO src/scheduler.rs:64: starting IO
2023-10-09T01:58:32.4821657Z ac1 waiting for inbox IDLE to become ready
2023-10-09T01:58:32.4822298Z 6.30 [events-ac1] INFO src/scheduler.rs:333: starting inbox loop
2023-10-09T01:58:32.4874502Z 6.30 [events-ac1] INFO src/job.rs:264: Loading job.
2023-10-09T01:58:32.4874957Z 6.30 [events-ac1] INFO src/scheduler.rs:649: starting smtp loop
2023-10-09T01:58:32.4875505Z 6.30 [events-ac1] INFO src/location.rs:676: Location loop is waiting for 24h 0m 0s or interrupt
2023-10-09T01:58:32.4876055Z 6.30 [events-ac1] INFO src/smtp.rs:651: Selected rows from SMTP queue: [].
2023-10-09T01:58:32.4876636Z 6.30 [events-ac1] INFO src/ephemeral.rs:567: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2023-10-09T01:58:32.4877160Z 6.30 [events-ac1] INFO src/scheduler.rs:812: scheduler is running
2023-10-09T01:58:32.4877697Z 6.30 [events-ac1] INFO src/contact.rs:1659: Recently seen loop waiting for 24h 0m 0s or interrupt
2023-10-09T01:58:32.4878180Z 6.30 [events-ac1] INFO src/smtp.rs:754: Sending MDNs
2023-10-09T01:58:32.4878614Z 6.30 [events-ac1] INFO src/scheduler.rs:688: smtp fake idle - started
2023-10-09T01:58:32.4879072Z 6.30 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4879607Z 6.30 [events-ac1] INFO src/scheduler.rs:710: smtp has no messages to retry, waiting for interrupt
2023-10-09T01:58:32.4880106Z 6.30 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4880555Z 6.30 [events-ac1] INFO src/imap.rs:309: Connecting to IMAP server
2023-10-09T01:58:32.4881388Z 6.30 [events-ac1] INFO src/net.rs:70: Resolved mail.ci.testrun.org:993 into 65.109.227.209:993.
2023-10-09T01:58:32.4882095Z 6.30 [events-ac1] INFO src/net.rs:70: Resolved mail.ci.testrun.org:993 into [2a01:4f9:c010:8f89::1]:993.
2023-10-09T01:58:32.4882617Z 6.99 [events-ac1] INFO src/imap.rs:378: Logging into IMAP server with LOGIN
2023-10-09T01:58:32.4883161Z 8.83 [events-ac1] DC_EVENT_IMAP_CONNECTED data1=0 data2=IMAP-LOGIN as 7dzs8@ci.testrun.org
2023-10-09T01:58:32.4883658Z 8.83 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4884143Z 8.83 [events-ac1] INFO src/imap.rs:395: Successfully logged into IMAP server
2023-10-09T01:58:32.4884831Z 9.04 [events-ac1] INFO src/imap.rs:887: 0 mails read from "INBOX".
2023-10-09T01:58:32.4885311Z 9.04 [events-ac1] INFO src/imap/scan_folders.rs:27: Starting full folder scan
2023-10-09T01:58:32.4885797Z 9.26 [events-ac1] INFO src/imap.rs:631: Folder "DeltaChat" is empty.
2023-10-09T01:58:32.4886285Z 9.26 [events-ac1] INFO src/imap.rs:719: No new emails in folder "DeltaChat".
2023-10-09T01:58:32.4894182Z 9.47 [events-ac1] INFO src/imap.rs:631: Folder "Archive" is empty.
2023-10-09T01:58:32.4894670Z 9.47 [events-ac1] INFO src/imap.rs:719: No new emails in folder "Archive".
2023-10-09T01:58:32.4895142Z 9.61 [events-ac1] INFO src/imap.rs:631: Folder "Sent" is empty.
2023-10-09T01:58:32.4895607Z 9.61 [events-ac1] INFO src/imap.rs:719: No new emails in folder "Sent".
2023-10-09T01:58:32.4896073Z 9.73 [events-ac1] INFO src/imap.rs:631: Folder "Junk" is empty.
2023-10-09T01:58:32.4896537Z 9.73 [events-ac1] INFO src/imap.rs:719: No new emails in folder "Junk".
2023-10-09T01:58:32.4896997Z 9.96 [events-ac1] INFO src/imap.rs:887: 0 mails read from "INBOX".
2023-10-09T01:58:32.4897439Z 10.08 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4897892Z 10.08 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
2023-10-09T01:58:32.4898262Z 10.08 [MAIN-ac1] inbox IDLE ready
2023-10-09T01:58:32.4898729Z 10.08 [events-ac1] INFO src/scheduler.rs:577: IMAP session supports IDLE, using it.
2023-10-09T01:58:32.4899273Z 10.20 [events-ac1] INFO src/imap/idle.rs:59: INBOX: Idle entering wait-on-remote state
2023-10-09T01:58:32.4899746Z 14.47 [events-ac2] INFO src/scheduler.rs:64: starting IO
2023-10-09T01:58:32.4900071Z ac2 waiting for inbox IDLE to become ready
2023-10-09T01:58:32.4900498Z 14.47 [events-ac2] INFO src/scheduler.rs:333: starting inbox loop
2023-10-09T01:58:32.4901185Z 14.47 [events-ac2] INFO src/job.rs:264: Loading job.
2023-10-09T01:58:32.4908885Z 14.47 [events-ac2] INFO src/scheduler.rs:649: starting smtp loop
2023-10-09T01:58:32.4909369Z 14.47 [events-ac2] INFO src/scheduler.rs:812: scheduler is running
2023-10-09T01:58:32.4909914Z 14.47 [events-ac2] INFO src/location.rs:676: Location loop is waiting for 24h 0m 0s or interrupt
2023-10-09T01:58:32.4910489Z 14.47 [events-ac2] INFO src/contact.rs:1659: Recently seen loop waiting for 24h 0m 0s or interrupt
2023-10-09T01:58:32.4911020Z 14.47 [events-ac2] INFO src/smtp.rs:651: Selected rows from SMTP queue: [].
2023-10-09T01:58:32.4911474Z 14.47 [events-ac2] INFO src/smtp.rs:754: Sending MDNs
2023-10-09T01:58:32.4911920Z 14.47 [events-ac2] INFO src/scheduler.rs:688: smtp fake idle - started
2023-10-09T01:58:32.4912382Z 14.47 [events-ac2] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4912920Z 14.47 [events-ac2] INFO src/scheduler.rs:710: smtp has no messages to retry, waiting for interrupt
2023-10-09T01:58:32.4913691Z 14.47 [events-ac2] INFO src/ephemeral.rs:567: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2023-10-09T01:58:32.4914200Z 14.47 [events-ac2] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4914657Z 14.47 [events-ac2] INFO src/imap.rs:309: Connecting to IMAP server
2023-10-09T01:58:32.4915188Z 14.47 [events-ac2] INFO src/net.rs:70: Resolved mail.ci.testrun.org:993 into 65.109.227.209:993.
2023-10-09T01:58:32.4919463Z 14.47 [events-ac2] INFO src/net.rs:70: Resolved mail.ci.testrun.org:993 into [2a01:4f9:c010:8f89::1]:993.
2023-10-09T01:58:32.4920496Z 15.29 [events-ac2] INFO src/imap.rs:378: Logging into IMAP server with LOGIN
2023-10-09T01:58:32.4921385Z 17.24 [events-ac2] DC_EVENT_IMAP_CONNECTED data1=0 data2=IMAP-LOGIN as m25dm@ci.testrun.org
2023-10-09T01:58:32.4922300Z 17.24 [events-ac2] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4922998Z 17.24 [events-ac2] INFO src/imap.rs:395: Successfully logged into IMAP server
2023-10-09T01:58:32.4974094Z 17.45 [events-ac2] INFO src/imap.rs:887: 0 mails read from "INBOX".
2023-10-09T01:58:32.4974615Z 17.45 [events-ac2] INFO src/imap/scan_folders.rs:27: Starting full folder scan
2023-10-09T01:58:32.4975111Z 17.67 [events-ac2] INFO src/imap.rs:631: Folder "DeltaChat" is empty.
2023-10-09T01:58:32.4975594Z 17.67 [events-ac2] INFO src/imap.rs:719: No new emails in folder "DeltaChat".
2023-10-09T01:58:32.4976070Z 17.81 [events-ac2] INFO src/imap.rs:631: Folder "Archive" is empty.
2023-10-09T01:58:32.4976547Z 17.81 [events-ac2] INFO src/imap.rs:719: No new emails in folder "Archive".
2023-10-09T01:58:32.4977020Z 17.94 [events-ac2] INFO src/imap.rs:631: Folder "Sent" is empty.
2023-10-09T01:58:32.4977483Z 17.94 [events-ac2] INFO src/imap.rs:719: No new emails in folder "Sent".
2023-10-09T01:58:32.4985060Z 18.06 [events-ac2] INFO src/imap.rs:631: Folder "Junk" is empty.
2023-10-09T01:58:32.4985540Z 18.06 [events-ac2] INFO src/imap.rs:719: No new emails in folder "Junk".
2023-10-09T01:58:32.4986024Z 18.28 [events-ac2] INFO src/imap.rs:887: 0 mails read from "INBOX".
2023-10-09T01:58:32.4986477Z 18.39 [events-ac2] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4986923Z 18.39 [events-ac2] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
2023-10-09T01:58:32.4987432Z 18.39 [events-ac2] INFO src/scheduler.rs:577: IMAP session supports IDLE, using it.
2023-10-09T01:58:32.4987842Z 18.39 [MAIN-ac2] inbox IDLE ready
2023-10-09T01:58:32.4988656Z finished, account2state {<Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac1/dc.db>: 'IDLEREADY', <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac2/dc.db>: 'IDLEREADY'}
2023-10-09T01:58:32.4989319Z all accounts online
2023-10-09T01:58:32.4989463Z 
2023-10-09T01:58:32.4989613Z ========== create unpromoted group chat ==========
2023-10-09T01:58:32.4989816Z 
2023-10-09T01:58:32.4990057Z ==========18.39 [events-ac1] DC_EVENT_MSGS_CHANGED data1=0 data2=0
2023-10-09T01:58:32.4990671Z  ac1: set profile image on unpromoted chat ==========
2023-10-09T01:58:32.4991145Z 18.39 [events-ac1] DC_EVENT_NEW_BLOB_FILE data1=0 data2=$BLOBDIR/d.png
2023-10-09T01:58:32.4991591Z 18.39 [events-ac1] DC_EVENT_CHAT_MODIFIED data1=12 data2=0
2023-10-09T01:58:32.4991798Z 
2023-10-09T01:58:32.4999006Z ========== ac1: send text to promote chat (XXX without contact added) ==========
2023-10-09T01:58:32.4999683Z 18.39 [events-ac1] INFO src/ephemeral.rs:567: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2023-10-09T01:58:32.5000280Z 18.39 [events-ac1] INFO src/chat.rs:2426: Message Msg#12 has no recipient, skipping smtp-send.
2023-10-09T01:58:32.5000781Z 18.39 [events-ac1] DC_EVENT_MSG_DELIVERED data1=12 data2=12
2023-10-09T01:58:32.5000989Z 
2023-10-09T01:58:32.5001150Z ========== ac2: check that initial message arrived ==========
2023-10-09T01:58:32.5001654Z 18.39 [events-ac2] INFO src/contact.rs:737: added contact id=10 addr=7dzs8@ci.testrun.org
2023-10-09T01:58:32.5002134Z 18.39 [events-ac2] DC_EVENT_CONTACTS_CHANGED data1=10 data2=0
2023-10-09T01:58:32.5002556Z 18.39 [events-ac2] DC_EVENT_MSGS_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5002758Z 
2023-10-09T01:58:32.5002914Z ========== ac1: add ac2 to promoted group chat ==========
2023-10-09T01:58:32.5003390Z 18.39 [events-ac1] INFO src/contact.rs:737: added contact id=10 addr=m25dm@ci.testrun.org
2023-10-09T01:58:32.5003861Z 18.39 [events-ac1] DC_EVENT_CONTACTS_CHANGED data1=10 data2=0
2023-10-09T01:58:32.5004360Z 18.39 [events-ac1] INFO src/chat.rs:1183: Set gossiped_timestamp for chat Chat#12 to 0.
2023-10-09T01:58:32.5005524Z 18.39 [events-ac1] INFO src/ephemeral.rs:567: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2023-10-09T01:58:32.5012478Z 18.39 [events-ac1] INFO src/mimefactory.rs:1098: setting group image '$BLOBDIR/d.png'
2023-10-09T01:58:32.5013084Z 18.39 [events-ac1] INFO src/e2ee.rs:85: peerstate for "m25dm@ci.testrun.org" missing, cannot encrypt
2023-10-09T01:58:32.5013579Z 18.39 [events-ac1] DC_EVENT_MSGS_CHANGED data1=12 data2=13
2023-10-09T01:58:32.5013771Z 
2023-10-09T01:58:32.5013932Z ========== ac1: send a first message to ac2 ==========
2023-10-09T01:58:32.5014389Z 18.40 [events-ac1] INFO src/scheduler.rs:714: smtp fake idle - interrupted
2023-10-09T01:58:32.5014890Z 18.40 [events-ac1] INFO src/smtp.rs:651: Selected rows from SMTP queue: [1].
2023-10-09T01:58:32.5015363Z 18.40 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5015795Z 18.40 [events-ac1] DC_EVENT_CHAT_MODIFIED data1=12 data2=0
2023-10-09T01:58:32.5016312Z 18.40 [events-ac1] INFO src/net.rs:70: Resolved mail.ci.testrun.org:465 into 65.109.227.209:465.
2023-10-09T01:58:32.5017008Z 18.40 [events-ac1] INFO src/net.rs:70: Resolved mail.ci.testrun.org:465 into [2a01:4f9:c010:8f89::1]:465.
2023-10-09T01:58:32.5017617Z 18.40 [events-ac1] INFO src/ephemeral.rs:567: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2023-10-09T01:58:32.5018228Z 18.40 [events-ac1] INFO src/e2ee.rs:85: peerstate for "m25dm@ci.testrun.org" missing, cannot encrypt
2023-10-09T01:58:32.5018723Z 18.40 [events-ac1] DC_EVENT_MSGS_CHANGED data1=12 data2=14
2023-10-09T01:58:32.5018930Z 
2023-10-09T01:58:32.5025550Z ========== ac2: wait for receiving message from ac1 ==========
2023-10-09T01:58:32.5026129Z 18.50 [events-ac2] INFO src/imap/idle.rs:59: INBOX: Idle entering wait-on-remote state
2023-10-09T01:58:32.5026711Z 20.89 [events-ac1] DC_EVENT_SMTP_CONNECTED data1=0 data2=SMTP-LOGIN as 7dzs8@ci.testrun.org ok
2023-10-09T01:58:32.5027274Z 20.89 [events-ac1] INFO src/smtp.rs:564: Try number 1 to send message Msg#13 (entry 1) over SMTP
2023-10-09T01:58:32.5027788Z 20.89 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5028325Z 21.74 [events-ac1] INFO src/smtp/send.rs:70: Message len=5644 was SMTP-sent to m25dm@ci.testrun.org.
2023-10-09T01:58:32.5028950Z 21.74 [events-ac1] DC_EVENT_SMTP_MESSAGE_SENT data1=0 data2=Message len=5644 was SMTP-sent to m25dm@ci.testrun.org
2023-10-09T01:58:32.5029452Z 21.74 [events-ac1] DC_EVENT_MSG_DELIVERED data1=12 data2=13
2023-10-09T01:58:32.5030094Z 21.74 [events-ac1] INFO src/smtp.rs:754: Sending MDNs
2023-10-09T01:58:32.5030559Z 21.74 [events-ac1] INFO src/scheduler.rs:688: smtp fake idle - started
2023-10-09T01:58:32.5031008Z 21.74 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5031551Z 21.74 [events-ac1] INFO src/scheduler.rs:710: smtp has no messages to retry, waiting for interrupt
2023-10-09T01:58:32.5034933Z 21.74 [events-ac1] INFO src/scheduler.rs:714: smtp fake idle - interrupted
2023-10-09T01:58:32.5035695Z 21.74 [events-ac1] INFO src/smtp.rs:651: Selected rows from SMTP queue: [2].
2023-10-09T01:58:32.5036470Z 21.74 [events-ac1] INFO src/smtp.rs:564: Try number 1 to send message Msg#14 (entry 2) over SMTP
2023-10-09T01:58:32.5037163Z 21.74 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5037899Z 22.71 [events-ac1] INFO src/smtp/send.rs:70: Message len=1439 was SMTP-sent to m25dm@ci.testrun.org.
2023-10-09T01:58:32.5038717Z 22.71 [events-ac1] DC_EVENT_SMTP_MESSAGE_SENT data1=0 data2=Message len=1439 was SMTP-sent to m25dm@ci.testrun.org
2023-10-09T01:58:32.5039368Z 22.71 [events-ac1] DC_EVENT_MSG_DELIVERED data1=12 data2=14
2023-10-09T01:58:32.5039863Z 22.71 [events-ac1] INFO src/smtp.rs:754: Sending MDNs
2023-10-09T01:58:32.5040394Z 22.71 [events-ac1] INFO src/scheduler.rs:688: smtp fake idle - started
2023-10-09T01:58:32.5040936Z 22.71 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5041567Z 22.71 [events-ac1] INFO src/scheduler.rs:710: smtp has no messages to retry, waiting for interrupt
2023-10-09T01:58:32.5042621Z 23.60 [events-ac2] INFO src/imap/idle.rs:74: INBOX: Idle has NewData ResponseData { raw: 4096, response: MailboxData(Exists(2)) }
2023-10-09T01:58:32.5043230Z 23.71 [events-ac2] INFO src/job.rs:264: Loading job.
2023-10-09T01:58:32.5053043Z 23.84 [events-ac2] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2023-10-09T01:58:32.5053796Z 23.84 [events-ac2] INFO src/imap.rs:1409: Starting a full FETCH of message set "30:31".
2023-10-09T01:58:32.5054511Z 23.99 [events-ac2] INFO src/imap.rs:1527: Passing message UID 30 to receive_imf().
2023-10-09T01:58:32.5055221Z 23.99 [events-ac2] INFO src/receive_imf.rs:103: Receiving message, seen=false...
2023-10-09T01:58:32.5056051Z 23.99 [events-ac2] INFO src/receive_imf.rs:142: Received message has Message-Id: Gr.roiLi58TsyW.X5-SeyxtbIH@ci.testrun.org
2023-10-09T01:58:32.5056908Z 23.99 [events-ac2] INFO src/chat.rs:290: Created group/mailinglist 'hello' grpid=roiLi58TsyW as Chat#13, blocked=Not.
2023-10-09T01:58:32.5057579Z 23.99 [events-ac2] DC_EVENT_CHAT_MODIFIED data1=13 data2=0
2023-10-09T01:58:32.5058237Z 24.00 [events-ac2] INFO src/receive_imf.rs:1839: Recreating chat Chat#13 member list with {ContactId(1), ContactId(10)}.
2023-10-09T01:58:32.5058931Z 24.00 [events-ac2] INFO src/receive_imf.rs:1278: Message has 1 parts and is assigned to chat #Chat#13.
2023-10-09T01:58:32.5059608Z 24.00 [events-ac2] INFO src/contact.rs:1659: Recently seen loop waiting for 0h 9m 57s or interrupt
2023-10-09T01:58:32.5060187Z 24.00 [events-ac2] DC_EVENT_INCOMING_MSG data1=13 data2=12
2023-10-09T01:58:32.5069539Z --------------------------- Captured stdout teardown ---------------------------
2023-10-09T01:58:32.5070266Z 24.23 [events-ac2] INFO src/imap/idle.rs:59: INBOX: Idle entering wait-on-remote state
2023-10-09T01:58:32.5071175Z 28.13 [events-ac2] INFO src/imap/idle.rs:74: INBOX: Idle has NewData ResponseData { raw: 4096, response: Fetch(1, [ModSeq(65), Flags(["\\Seen", "\\Recent"])]) }
2023-10-09T01:58:32.5071798Z ===============  ===============
2023-10-09T01:58:32.5072248Z ARCH=64 AUTHSERV_ID_CANDIDATES= BCC_SELF=0 
2023-10-09T01:58:32.5072927Z BLOBDIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac1/dc.db-blobs 
2023-10-09T01:58:32.5073511Z BOT=0 CONFIGURED_INBOX_FOLDER=INBOX CONFIGURED_MVBOX_FOLDER=<unset> 
2023-10-09T01:58:32.5074024Z CONFIGURED_SENTBOX_FOLDER=Sent CONFIGURED_TRASH_FOLDER=Trash 
2023-10-09T01:58:32.5074883Z DATABASE_DIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac1/dc.db 
2023-10-09T01:58:32.5075430Z DATABASE_ENCRYPTED=false DATABASE_VERSION=101 DEBUG_LOGGING=0 
2023-10-09T01:58:32.5075886Z DELETE_DEVICE_AFTER=0 DELETE_SERVER_AFTER=0 DELETE_TO_TRASH=<unset> 
2023-10-09T01:58:32.5076386Z DELTACHAT_CORE_VERSION=v1.124.1 DISPLAY_NAME=0 DOWNLOAD_LIMIT=0 E2EE_ENABLED=1 
2023-10-09T01:58:32.5086546Z ENTERED_ACCOUNT_SETTINGS=7dzs8@ci.testrun.org imap:unset:***:unset:0:Automatic:cert_strict:AUTH_NORMAL smtp:unset:0:unset:0:Automatic:cert_strict:AUTH_NORMAL 
2023-10-09T01:58:32.5087312Z FETCH_EXISTING_MSGS=0 FETCHED_EXISTING_MSGS=true 
2023-10-09T01:58:32.5087787Z FINGERPRINT=2E6FA2CB23B532D728634B5864B08F61A9ED9443 FOLDERS_CONFIGURED=4 
2023-10-09T01:58:32.5088284Z IMAP_SERVER_ID={"name": "Dovecot"} IS_CONFIGURED=1 JOURNAL_MODE=wal 
2023-10-09T01:58:32.5088767Z KEY_GEN_TYPE=0 LAST_HOUSEKEEPING=1696816365 LAST_MSG_ID=0 LEVEL=awesome 
2023-10-09T01:58:32.5089259Z MDNS_ENABLED=1 MEDIA_QUALITY=0 MESSAGES_IN_CONTACT_REQUESTS=0 MVBOX_MOVE=0 
2023-10-09T01:58:32.5089763Z NUM_CPUS=2 NUMBER_OF_CHAT_MESSAGES=5 NUMBER_OF_CHATS=3 NUMBER_OF_CONTACTS=1 
2023-10-09T01:58:32.5090267Z ONLY_FETCH_MVBOX=0 PRIVATE_KEY_COUNT=1 PUBLIC_KEY_COUNT=0 QUOTA_EXCEEDING=0 
2023-10-09T01:58:32.5090741Z SCAN_ALL_FOLDERS_DEBOUNCE_SECS=60 SECONDARY_ADDRS= SELFAVATAR=<unset> 
2023-10-09T01:58:32.5091203Z SEND_SYNC_MSGS=0 SENTBOX_WATCH=0 SHOW_EMAILS=2 SIGN_UNENCRYPTED=0 
2023-10-09T01:58:32.5091657Z SOCKS5_ENABLED=0 SQLITE_VERSION=3.39.4 UPTIME=0h 0m 24s 
2023-10-09T01:58:32.5092319Z USED_ACCOUNT_SETTINGS=7dzs8@ci.testrun.org imap:7dzs8@ci.testrun.org:***:mail.ci.testrun.org:993:Ssl:cert_strict:AUTH_NORMAL smtp:7dzs8@ci.testrun.org:***:mail.ci.testrun.org:465:Ssl:cert_strict:AUTH_NORMAL 
2023-10-09T01:58:32.5093133Z WEBRTC_INSTANCE=<unset> 
2023-10-09T01:58:32.5097768Z --------- EMPTY FOLDERS: ['Junk', 'Sent', 'Archive', 'DeltaChat', 'Trash', 'Drafts', 'INBOX']
2023-10-09T01:58:32.5098197Z 
2023-10-09T01:58:32.5098377Z ===============  ===============
2023-10-09T01:58:32.5098749Z ARCH=64 AUTHSERV_ID_CANDIDATES= BCC_SELF=0 
2023-10-09T01:58:32.5099345Z BLOBDIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac2/dc.db-blobs 
2023-10-09T01:58:32.5099875Z BOT=0 CONFIGURED_INBOX_FOLDER=INBOX CONFIGURED_MVBOX_FOLDER=<unset> 
2023-10-09T01:58:32.5100350Z CONFIGURED_SENTBOX_FOLDER=Sent CONFIGURED_TRASH_FOLDER=Trash 
2023-10-09T01:58:32.5100953Z DATABASE_DIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_set_get_group_image0/ac2/dc.db 
2023-10-09T01:58:32.5101462Z DATABASE_ENCRYPTED=false DATABASE_VERSION=101 DEBUG_LOGGING=0 
2023-10-09T01:58:32.5101937Z DELETE_DEVICE_AFTER=0 DELETE_SERVER_AFTER=0 DELETE_TO_TRASH=<unset> 
2023-10-09T01:58:32.5102426Z DELTACHAT_CORE_VERSION=v1.124.1 DISPLAY_NAME=0 DOWNLOAD_LIMIT=0 E2EE_ENABLED=1 
2023-10-09T01:58:32.5103055Z ENTERED_ACCOUNT_SETTINGS=m25dm@ci.testrun.org imap:unset:***:unset:0:Automatic:cert_strict:AUTH_NORMAL smtp:unset:0:unset:0:Automatic:cert_strict:AUTH_NORMAL 
2023-10-09T01:58:32.5103621Z FETCH_EXISTING_MSGS=0 FETCHED_EXISTING_MSGS=true 
2023-10-09T01:58:32.5104073Z FINGERPRINT=CCCB5AA9F6E1141C943165F1DB18B18CBCF70487 FOLDERS_CONFIGURED=4 
2023-10-09T01:58:32.5104566Z IMAP_SERVER_ID={"name": "Dovecot"} IS_CONFIGURED=1 JOURNAL_MODE=wal 
2023-10-09T01:58:32.5108963Z KEY_GEN_TYPE=0 LAST_HOUSEKEEPING=1696816369 LAST_MSG_ID=0 LEVEL=awesome 
2023-10-09T01:58:32.5109557Z MDNS_ENABLED=1 MEDIA_QUALITY=0 MESSAGES_IN_CONTACT_REQUESTS=0 MVBOX_MOVE=0 
2023-10-09T01:58:32.5110067Z NUM_CPUS=2 NUMBER_OF_CHAT_MESSAGES=4 NUMBER_OF_CHATS=4 NUMBER_OF_CONTACTS=1 
2023-10-09T01:58:32.5110588Z ONLY_FETCH_MVBOX=0 PRIVATE_KEY_COUNT=1 PUBLIC_KEY_COUNT=1 QUOTA_EXCEEDING=0 
2023-10-09T01:58:32.5111086Z SCAN_ALL_FOLDERS_DEBOUNCE_SECS=60 SECONDARY_ADDRS= SELFAVATAR=<unset> 
2023-10-09T01:58:32.5111564Z SEND_SYNC_MSGS=0 SENTBOX_WATCH=0 SHOW_EMAILS=2 SIGN_UNENCRYPTED=0 
2023-10-09T01:58:32.5112020Z SOCKS5_ENABLED=0 SQLITE_VERSION=3.39.4 UPTIME=0h 0m 25s 
2023-10-09T01:58:32.5112891Z USED_ACCOUNT_SETTINGS=m25dm@ci.testrun.org imap:m25dm@ci.testrun.org:***:mail.ci.testrun.org:993:Ssl:cert_strict:AUTH_NORMAL smtp:m25dm@ci.testrun.org:***:mail.ci.testrun.org:465:Ssl:cert_strict:AUTH_NORMAL 
2023-10-09T01:58:32.5113493Z WEBRTC_INSTANCE=<unset> 
2023-10-09T01:58:32.5113950Z --------- INBOX 2 messages ---------
2023-10-09T01:58:32.5114311Z Message 30 has empty body
2023-10-09T01:58:32.5114655Z Message 31 has empty body
2023-10-09T01:58:32.5115155Z --------- EMPTY FOLDERS: ['Junk', 'Sent', 'Archive', 'DeltaChat', 'Trash', 'Drafts']
2023-10-09T01:58:32.5115477Z 
2023-10-09T01:58:32.5115558Z 
2023-10-09T01:58:32.5115856Z 28.24 [events-ac2] INFO src/job.rs:264: Loading job.
2023-10-09T01:58:32.5116274Z 28.34 [MAIN-ac2] stop_ongoing
2023-10-09T01:58:32.5116727Z 28.34 [MAIN-ac2] dc_stop_io (stop core IO scheduler)
2023-10-09T01:58:32.5121226Z 28.34 [events-ac2] EVENT THREAD FINISHED
2023-10-09T01:58:32.5121790Z 28.34 [MAIN-ac2] wait for event thread to finish
2023-10-09T01:58:32.5122340Z 28.34 [MAIN-ac2] remove dc_context references, making the Account unusable
2023-10-09T01:58:32.5122831Z 28.34 [MAIN-ac2] shutdown finished
2023-10-09T01:58:32.5123343Z 28.46 [MAIN-ac1] stop_ongoing
2023-10-09T01:58:32.5123970Z 28.46 [events-ac1] EVENT THREAD FINISHED
2023-10-09T01:58:32.5124759Z 28.46 [MAIN-ac1] dc_stop_io (stop core IO scheduler)
2023-10-09T01:58:32.5125251Z 28.46 [MAIN-ac1] wait for event thread to finish
2023-10-09T01:58:32.5125798Z 28.46 [MAIN-ac1] remove dc_context references, making the Account unusable
2023-10-09T01:58:32.5126277Z 28.46 [MAIN-ac1] shutdown finished
2023-10-09T01:58:32.5126917Z ================== 1 failed, 175 passed in 355.82s (0:05:55) ===================
```

Taken from https://github.com/deltachat/deltachat-core-rust/actions/runs/6451399099/job/17512082648?pr=4804

--- a/python/tests/test_1_online.py
+++ b/python/tests/test_1_online.py
@@ -1926,13 +1926,15 @@ def test_set_get_group_image(acfactory, data, lp):
     lp.sec("ac1: add ac2 to promoted group chat")
     chat.add_contact(ac2)  # sends one message
 
+    lp.sec("ac2: wait for receiving member added message from ac1")
+    msg1 = ac2._evtracker.wait_next_incoming_message()
+    assert msg1.is_system_message()  # Member added
+
     lp.sec("ac1: send a first message to ac2")
     chat.send_text("hi")  # sends another message
     assert chat.is_promoted()
 
     lp.sec("ac2: wait for receiving message from ac1")
-    msg1 = ac2._evtracker.wait_next_incoming_message()
-    assert msg1.is_system_message()  # Member added
     msg2 = ac2._evtracker.wait_next_incoming_message()
     assert msg2.text == "hi"
     assert msg1.chat.id == msg2.chat.id

